src/utils/types.d.ts

```diff
@@ -0,0 +1,6 @@
+// https://github.com/microsoft/TypeScript/issues/29729#issuecomment-505826972
+//TODO use version of type-fest instead?
```

Comment:
remove comment

Reply:
will do after other PR done and update this branch